services/blockchain-connector/tests/functional/encoder.test.js

```diff
@@ -34,21 +40,33 @@ const broker = new ServiceBroker({
 	logger: console,
 });
 
-xdescribe('Functional tests for encoder', () => {
-	beforeAll(() => broker.start());
-	afterAll(() => broker.stop());
+beforeAll(() => broker.start());
+afterAll(() => broker.stop());
 
-	it('encode transaction', async () => {
+xdescribe('Encode transaction', () => {
```

Comment:
This test case was failing in Jenkins so we skipped the test with Update Jenkinsfile PR. It can be fixed and enable with the issue https://github.com/LiskHQ/lisk-service/issues/1491